### PR TITLE
version: Update versions of linux kernel and qemu-lite

### PR DESCRIPTION
**Pull request by gorozco1**
qemu-lite and linux-kernel used to run clear-containers were pointing to
the wrong versions. Currently we are using newer versions of linux
kernel and qemu-lite.

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -7,10 +7,10 @@ docker_ubuntu_version=16.04
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
 clear_vm_image_version=14700
-clear_vm_kernel_version=4.5-57
+clear_vm_kernel_version=4.9.4-57
```
Comment (jcvenegas) [inline versions.txt]: looks good, btw we need to update kernel version to the last minior version (4.9.24)

**Comment by jcvenegas** [comment]
lgtm

File: versions.txt (hunk 1 above)
Comment (gorozco1) [inline versions.txt]: At the moment the kernel we use and we build is 4.9.4 no 4.9.24, when we update the kernel we can change this line ;-)

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by jodh-intel** [comment]
FYI - I've raised #847 for an idea I have to improve our process and avoid such oversights.

lgtm.

File: versions.txt (hunk 1 above)
Comment (grahamwhaley) [inline versions.txt]: Note, I have recently asked @miguelinux if/when/should we update the container kernel to the latest longerm (4.9.24)... I think that is now overdue.

**Pull request merged**